src/experimental/tools/mediaCapabilitiesProber/probers/HDCPPolicy.ts

```diff
@@ -29,24 +28,18 @@ export type IMediaKeyStatus =
   | "internal-error";
 
 /**
- * @param {Object} config
+ * @param {string} hdcpVersion
  * @returns {Promise}
  */
-export default function probeHDCPPolicy(
-  config: IMediaConfiguration,
-): Promise<[ProberStatus]> {
+export default async function probeHDCPPolicy(
```

Comment:
I feel that this function can either throw or return "NotSupported"
I'm wondering if we should catch any errors and return "NotSupported"

Reply:
Complex to say. 

It fails if EME API failed to be called.

`NotSupported` would be if there's a high priority of the HDCP version being unsupported but here we're not so sure.

Maybe `Unknown`? I'm under the impression that for now `Unknown` is only for when the `getStatusForPolicy` API is not available, which is slightly different than e.g. `requestMediaKeySystemAccess` failing yet both would just indicate that it is unknown.....

Then, when applications handle promises, they generally also handle rejections (even linters often force that). So I'm not sure.

Reply:
I removed the Unkown and it only returns "Supported" / "NotSupported" or reject